crates/wasmtime/src/config.rs

```diff
@@ -189,13 +190,15 @@ impl Config {
             memory_init_cow: true,
             memory_guaranteed_dense_image_size: 16 << 20,
             force_memory_init_memfd: false,
+            function_references: false,
```

Comment:
This won't actually be stored as a field in `Config` but rather `self.features`, so this field can be omitted from the Wasmtime crate.